commands/project/transfer/project_transfer.go

```diff
@@ -0,0 +1,91 @@
+package transfer
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+func NewCmdTransfer(f *cmdutils.Factory) *cobra.Command {
+	var repoTransferCmd = &cobra.Command{
+		Use:   "transfer <command> [flags]",
```

Comment:
instead of `<command>`, it should be `<repo>`.

nit: I'm also thinking we should make it optional `[repo]` so that if `repo` is not provided, we get the repo from the current git directory (with `f.BaseRepo()`)